rosdep/python.yaml

```
@@ -8391,6 +8391,14 @@ python3-twisted:
     '*': ['python%{python3_pkgversion}-twisted']
     '7': null
   ubuntu: [python3-twisted]
+python3-tz:
+  alpine: [py3-tz]
+  arch: [python-pytz]
+  debian: [python3-tz]
+  fedora: [python3-pytz]
+  gentoo: [dev-python/pytz]
+  nixos: [python39Packages.pytz]
+  ubuntu: [python3-tz]
```

Hi, @airballking. Would you mind adding a rule for RHEL 8 as well?

Here's the package: https://repo.almalinux.org/almalinux/8/AppStream/x86_64/os/Packages/python3-pytz-2017.2-9.el8.noarch.rpm

```suggestion
  rhel:
    '*': [python3-pytz]
    '7': null
  ubuntu: [python3-tz]
```